docs/src/main/sphinx/admin/properties-writer-scaling.rst

```diff
@@ -19,8 +19,22 @@ the needs of the query.
 * **Type:** :ref:`prop-type-boolean`
 * **Default value:** ``true``
 
-Enable writer scaling. This can be specified on a per-query basis
-using the ``scale_writers`` session property.
+Enable writer scaling across different workers. In this, each worker will
+only get single writer task. So, the maximum scale it can reach is the
+total number of workers. This can be specified on a per-query basis using
+the ``scale_writers`` session property.
```

Comment:
```
Enable writer scaling by dynamically increasing the number of writer tasks on the cluster.
This can be specified on a per-query basis using the ``scale_writers`` session property.
```
The other two statements are not necessarily true in tardigrade mode.